src/go/cmd/app-rollout-controller/BUILD.bazel

```diff
@@ -39,13 +39,8 @@ pkg_tar(
 
 oci_image(
     name = "app-rollout-controller-image",
-    architecture = select({
```

Comment:
Can you add to the PR description why this change is needed (and correct)?

Reply:
'os', 'architecture' and 'variant' are taken from "base" and can not be overridden.